all/cli/pom.xml

```diff
@@ -5,7 +5,7 @@
    <parent>
       <groupId>org.infinispan</groupId>
       <artifactId>infinispan-all-parent</artifactId>
-      <version>9.2.2-SNAPSHOT</version>
+      <version>9.3.0-SNAHPSHOT</version>
```

Comment:
@tristantarrant Fat fingers, too much gnocco fritto!

Reply:
Damn! I'm using the fact it's Monday as my excuse for also missing this

Reply:
@tristantarrant Can you issue another PR using your new script please

Reply:
I just pushed a fix